Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1976,6 +1976,18 @@
   categories:
   - UplinkWearables
 
+#Sunrise-Start
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)